Header header
float64 position
float64 elapsed_time
FoodRegionData[] region_data_array 

